### ISSUE TITLE
Table header shrinks a little after each scroll. 

### ISSUE THREAD

**Issue by davidwiggy** [mkoryak/floatThead#294]
This isn't happening in Chrome, only in Firefox and IE. The plugin is build with position: absolute. Every thing works great on the initial scroll down. However, if the user scrolls down then back up and then back down repeatedly the header shrinks a little each time, depending on the width of the border. This is only happening when I add a border to the table. When I remove the border class the plugin behaves normally. 

I've build the table and add the code in jsfiddle to try and recreate the issue, however, in jsfiddle it is working fine. I'll provide the link anyway so that maybe you can help by examining the code. 

https://jsfiddle.net/wiggy1/bhefx9fa/

All update to date browsers.
Windows 7

Thanks for any help.

**Comment by mkoryak** [comment]
Thanks for the fiddle,  the html is probably fine except that using padding and cellspacing attributes isnt supported. There are css props that do the same thing that you should use. 

Perhaps the real table has a container and the content around it is "squeezing" it?

**Comment by davidwiggy** [comment]
I went ahead and just removed all padding and spacing attributes for testing purposes and this did not help. 

Please forgive my ignorance, but when you say real table are you referring to the main table that is holding all of the information? 

If this is what you are suggesting then you are correct. I tested this by removing the .mainTableContainer and aligning the real table to the left and setting the width to 50% and the problem is now resolved. 

However, I need the table centered in the page and without the .mainTableContainer the header floats to the left side of the screen when the plugin is activated. 

As here, https://jsfiddle.net/wiggy1/bhefx9fa/1/. 

Any suggestions for this. 

Also, thanks for the quick reply.

**Comment by mkoryak** [comment]
i wasnt trying to suggest a solution, but to point you in the direction i would take when trying to reproduce the issue. 

I cant really give you any solutions without seeing the actual issue.

Well, perhaps I can:

try setting a min-width on the container, preferably in pixels. also try width if min-width doesnt help. you can also try using position: 'fixed' and see if that makes any difference.

**Comment by mkoryak** [comment]
you can also try setting min-width or width on the table or on the container programatically before you init floatThead. 

There is a good chance that this is actually a bug in my lib, but I wont be able to fix it without a jsfiddle that reproduces it.

**Comment by davidwiggy** [comment]
mkoryak,

Thanks. I tried each of your suggestions individually, however, I had no success with any of them. 

But, I adjusted the width of the real table to 99% instead of 100% and this resolved the issue. This is interesting, just for testing, I set a border on the mainTableContainer and the real table with a width of 100 was slightly overflowing the div and every time the page scrolled up and down either the div shrunk or the table grew each time by the width of the smallBorder class.

Once I set the width of the real table to 99% the overflow problem stopped and this resolve the shrinking div or growing table. 

Once again thanks for you help with this matter.
